bucket/praat.json

```
@@ -21,15 +21,16 @@
         ]
     ],
     "checkver": {
-        "github": "https://github.com/praat/praat"
+        "url": "https://github.com/praat/praat/releases",
```

I do not see a reason to prefer url of all releaes instead of latest

```suggestion
        "github": "https://github.com/praat/praat",
```